ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -1701,6 +1703,31 @@ public sealed class ArchiWebHandler : IDisposable {
 			return result;
 		}
 
+		internal async Task<byte> GetCardCountForGame(uint appID) {
+			if (CachedCardCountsForGame.TryGetValue(appID, out byte result)) {
+				return result;
+			}
+
+			using IDocument? htmlDocument = await GetGameCardsPage(appID).ConfigureAwait(false);
+
+			if (htmlDocument == null) {
+				return 0;
+			}
+
+			List<IElement> htmlNodes = htmlDocument.SelectNodes("//div[@class='badge_card_set_cards']/div[starts-with(@class, 'badge_card_set_card')]");
+
+			if (htmlNodes.Count == 0) {
+				Bot.ArchiLogger.LogNullError(nameof(htmlNodes));
+
+				return 0;
+			}
+
+			result = (byte) htmlNodes.Count;
+			CachedCardCountsForGame.Add(appID, result);
```

Comment:
Will throw an exception if 2 bots will parse card count at the same time and appID will already exist. Also it's possible that 2 bots will call .Add at the same time, which can corrupt dictionary internal state, consider either locking on `CachedCardCountsForGame` or making it `ConcurrentDictionary`.

Reply:
Now I know what I wanted to do earlier :)